testing/btest/Baseline/scripts.base.protocols.redis.django-cloud/redis.log

```diff
@@ -0,0 +1,168 @@
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_35	10333147966386144929666651337523200000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_36	371993326789901217467999448150835200000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_37	13763753091226345046315979581580902400000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_38	523022617466601111760007224100074291200000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_39	20397882081197443358640281739902897356800000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_40	815915283247897734345611269596115894272000000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_41	33452526613163807108170062053440751665152000000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_42	1405006117752879898543142606244511569936384000000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_43	60415263063373835637355132068513997507264512000000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_44	2658271574788448768043625811014615890319638528000000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_45	119622220865480194561963161495657715064383733760000000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_46	5502622159812088949850305428800254892961651752960000000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_47	258623241511168180642964355153611979969197632389120000000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_48	12413915592536072670862289047373375038521486354677760000000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_49	608281864034267560872252163321295376887552831379210240000000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_50	30414093201713378043612608166064768844377641568960512000000000000	T	OK
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	SET	:1:factorial_50	30414093201713378043612608166064768844377641568960512000000000000	T	30414093201713378043612608166064768844377641568960512000000000000
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	GET	:1:factorial_50	-	T	30414093201713378043612608166064768844377641568960512000000000000
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	192.168.1.4	50044	18.234.186.95	10625	GET	:1:factorial_50	-	T	(empty)
```

Comment:
Just skimming, think there's something with the command/reply correlation off: Wireshark indicates `GET` returns a value and the `SET` above in line 63 doesn't return a value.

![Screenshot from 2025-05-24 15-50-33](https://github.com/user-attachments/assets/0b96ee1a-6a03-4f41-b0d4-fc057df01a6c)